spec/forms/editors_form_spec.rb

```diff
@@ -84,11 +94,12 @@
       before { allow(UserRegistrationService).to receive(:call).with(uid: access_id).and_return(nil) }
 
       it 'does not add the user and reports an error' do
-        expect(work.edit_users).to be_empty
+        expect(resource.edit_users).to be_empty
         form.save
-        expect(work.edit_users).to be_empty
+        expect(resource.edit_users).to be_empty
         expect(form.errors.full_messages).to contain_exactly(
-          'Edit users ' + I18n.t('dashboard.works.editors.not_found', access_id: access_id)
+          'Edit users ' +
+          I18n.t!('activemodel.errors.models.editors_form.attributes.edit_users.not_found', access_id: access_id)
```

Comment:
@awead here is a pro rails tip, in tests if you use the bang version `I18n.t!` then it will raise an error if the translation can't be found. We should probably do a sweep through everything in the spec directory and fix all these.

Reply:
Done! #865